lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -119,7 +119,6 @@ sub create_from_settings {
     my $result_source = $self->result_source;
     my $schema = $result_source->schema;
     my $job_settings = $schema->resultset('JobSettings');
-    my $txn_guard = $result_source->storage->txn_scope_guard;
```

Comment:
There are two places where we call create_from_settings, though. Will that other use case still work correctly?

Reply:
I just had a look and in lib/OpenQA/WebAPI/Controller/API/V1/Job.pm create_from_settings is also already called in a transaction from
```
$self->schema->txn_do(sub { $self->_create_jobs($global_params, $grouped_params) });
```
so maybe it's ok

Reply:
I took into account that the function is called from multiple places :-)